### PR TITLE
Addition of Content Distribution

### PR DESCRIPTION
**Pull request by mistryvinay**
New AppExample for Content Distribution,

- New menu option which demonstrates data being read by a Symphony chat bot - then being ingested by a websocket client in the form of the Market Flow UI

Dependencies are on these projects to be running to send data to the Market Flow UI
- https://github.com/SymphonyPlatformSolutions/symphony-content-distribution-bot-publisher
- https://github.com/SymphonyPlatformSolutions/symphony-content-distribution-bot-subscriber

### CODE REVIEW

**Comment by mistryvinay** [comment]
Note the preview isn't happy with ws instead of wss.  Will correct via this PR.

File: AppExamples/CleverDeal.React/src/Components/ContentDistribution/ContentDistribution.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import { Routes, Route, Link, useLocation } from "react-router-dom";
+import { LineChart, Line, XAxis, YAxis, Tooltip, ResponsiveContainer } from 'recharts';
+import { Search, Menu, TrendingUp, TrendingDown } from 'lucide-react';
+import MarketFeed from './MarketFeed';
+import MarketFlow from './MarketFlow';
+import './index.css';
```
Comment (antoinerollindev) [inline AppExamples/CleverDeal.React/src/Components/ContentDistribution/ContentDistribution.js:7]: this `index.css` needs to be removed as we inject it dynamically

Comment (mistryvinay) [reply AppExamples/CleverDeal.React/src/Components/ContentDistribution/ContentDistribution.js:7]: Just removed. Thanks Antoine

File: AppExamples/CleverDeal.React/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+  './src/Components/ContentDistribution/index.js',
+  './src/Components/ContentDistribution/ContentDistribution.js',
+  './src/Components/ContentDistribution/MarketFeed.js',
+  './src/Components/ContentDistribution/MarketFlow.js'
```
Comment (antoinerollindev) [inline AppExamples/CleverDeal.React/tailwind.config.js:7]: Could be simplified with `./src/Components/ContentDistribution/*.js`.

**Review by antoinerollindev — APPROVED** [review]
**Note for devs**

We didn't any better way to apply the tailwind css than adding the `require(index.css)` in the related components. If we do a classic import of the css file, it impacts all the app components.

As this is done in multiple components, it might be useful to extract this into a hook like (new hook.js file or something):
```
export const useTailwindCSS= (component) => {
    require('./index.css');
    return component;
};
```
And use it in component where it is required:
```
// ...
export default useTailwindCSS(MarketFeed);
```

File: AppExamples/CleverDeal.React/tailwind.config.js (hunk 2 above)
Comment (mistryvinay) [reply AppExamples/CleverDeal.React/tailwind.config.js:7]: Updated as recommended.

**Pull request closed without merging**